### PR TITLE
Added graph title

### PR DESCRIPTION
**Pull request by baljinderxd**
For #79

### CODE REVIEW

File: app/utils/prepareAnswer.js [hunk 1]
```diff
@@ -281,8 +281,8 @@ function dailyStatsGraph(body, n) {
   dateLabel.reverse();
 
   let link = `https://quickchart.io/chart?c={type:'line',data:{labels:[${dateLabel}],`;
-  link += `datasets:[{label:'Daily new cases',data:[${data}],backgroundColor:'white', fill:false, borderColor:'red'}]}}`;
-
+  link += `datasets:[{label:'Daily new cases',data:[${data}],backgroundColor:'white', fill:false, borderColor:'red'}]},`;
+  link += `options: {title:{display: true,text: 'Daily confirmed cases graphical analysis for past ${minN} days'},}}`;
```
Comment (gurrrung) [inline app/utils/prepareAnswer.js:285]: You don't have to use the word `graphical` because it's obvious.

File: app/utils/prepareAnswer.js [hunk 2]
```diff
@@ -302,8 +302,8 @@ function dailyActiveStatsGraph(body, n) {
   dateLabel.reverse();
 
   let link = `https://quickchart.io/chart?c={type:'line',data:{labels:[${dateLabel}],`;
-  link += `datasets:[{label:'Daily new active cases',data:[${data}],backgroundColor:'white', fill:false, borderColor:'blue'}]}}`;
-
+  link += `datasets:[{label:'Daily new active cases',data:[${data}],backgroundColor:'white', fill:false, borderColor:'blue'}]},`;
+  link += `options: {title:{display: true,text: 'Daily active cases graphical analysis for past ${minN} days'},}}`;
```
Comment (gurrrung) [inline app/utils/prepareAnswer.js:306]: same

File: app/utils/prepareAnswer.js [hunk 3]
```diff
@@ -321,7 +321,8 @@ function dailyRecStatsGraph(body, n) {
   dateLabel.reverse();
 
   let link = `https://quickchart.io/chart?c={type:'line',data:{labels:[${dateLabel}],`;
-  link += `datasets:[{label:'Daily new recovered cases',data:[${data}],backgroundColor:'white', fill:false, borderColor:'green'}]}}`;
+  link += `datasets:[{label:'Daily new recovered cases',data:[${data}],backgroundColor:'white', fill:false, borderColor:'green'}]},`;
```
Comment (gurrrung) [inline app/utils/prepareAnswer.js:324]: same

File: app/utils/prepareAnswer.js [hunk 4]
```diff
@@ -340,7 +341,8 @@ function dailyDecStatsGraph(body, n) {
   dateLabel.reverse();
 
   let link = `https://quickchart.io/chart?c={type:'line',data:{labels:[${dateLabel}],`;
-  link += `datasets:[{label:'Daily new deceased cases',data:[${data}],backgroundColor:'white', fill:false, borderColor:'grey'}]}}`;
+  link += `datasets:[{label:'Daily new deceased cases',data:[${data}],backgroundColor:'white', fill:false, borderColor:'grey'}]},`;
+  link += `options: {title:{display: true,text: 'Daily deceased cases graphical analysis for past ${minN} days'},}}`;
```
Comment (gurrrung) [inline app/utils/prepareAnswer.js:345]: same

File: app/utils/prepareAnswer.js [hunk 5]
```diff
@@ -371,7 +373,8 @@ function summaryGraph(body, n) {
   link += `datasets:[{label:'Confirmed',data:[${confirmed}],backgroundColor:'white', fill:false, borderColor:'red'},`;
   link += `{label:'Active',data:[${active}],backgroundColor:'white', fill:false, borderColor:'blue'},`;
   link += `{label:'Recovered',data:[${recovered}],backgroundColor:'white', fill:false, borderColor:'green'},`;
-  link += `{label:'Deceased',data:[${deaths}],backgroundColor:'white', fill:false, borderColor:'grey'}]}}`;
+  link += `{label:'Deceased',data:[${deaths}],backgroundColor:'white', fill:false, borderColor:'grey'}]},`;
+  link += `options: {title:{display: true,text: 'Daily cases graphical analysis for past ${minN} days'},}}`;
```
Comment (gurrrung) [inline app/utils/prepareAnswer.js:377]: same

**Review by gurrrung — CHANGES_REQUESTED** [review]
PTAL

File: app/utils/prepareAnswer.js (hunk 1 above)
Comment (baljinderxd) [reply app/utils/prepareAnswer.js:285]: Okay will remove it 👍

Comment (gurrrung) [reply app/utils/prepareAnswer.js:285]: thanks

File: app/utils/prepareAnswer.js (hunk 3 above)
Comment (gurrrung) [inline app/utils/prepareAnswer.js:324]: `g daily rec` gives 400 Bad request error in logs.

**Review by gurrrung — CHANGES_REQUESTED** [review]
PTAL

**Comment by gurrrung** [comment]
I got this:

```
Unhandled rejection Error: ETELEGRAM: 400 Bad Request: wrong file identifier/HTTP URL specified
    at /home/gurrrung/fun/CovidBot/node_modules/node-telegram-bot-api/src/telegram.js:283:15
    at tryCatcher (/home/gurrrung/fun/CovidBot/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/gurrrung/fun/CovidBot/node_modules/bluebird/js/release/promise.js:547:31)
    at Promise._settlePromise (/home/gurrrung/fun/CovidBot/node_modules/bluebird/js/release/promise.js:604:18)
    at Promise._settlePromise0 (/home/gurrrung/fun/CovidBot/node_modules/bluebird/js/release/promise.js:649:10)
    at Promise._settlePromises (/home/gurrrung/fun/CovidBot/node_modules/bluebird/js/release/promise.js:729:18)
    at _drainQueueStep (/home/gurrrung/fun/CovidBot/node_modules/bluebird/js/release/async.js:93:12)
    at _drainQueue (/home/gurrrung/fun/CovidBot/node_modules/bluebird/js/release/async.js:86:9)
    at Async._drainQueues (/home/gurrrung/fun/CovidBot/node_modules/bluebird/js/release/async.js:102:5)
    at Immediate.Async.drainQueues [as _onImmediate] (/home/gurrrung/fun/CovidBot/node_modules/bluebird/js/release/async.js:15:14)
    at processImmediate (internal/timers.js:456:21)


If you are able to figure out let me know if not just undo label for this command and we can merge this and create a new issue for this bug.

**Comment by gurrrung** [comment]
`graph 180` also gives error.

**Comment by baljinderxd** [comment]
> I got this:
> 
> ```
> Unhandled rejection Error: ETELEGRAM: 400 Bad Request: wrong file identifier/HTTP URL specified

Are you getting this for `graph` command?

> `graph 180` also gives error.

Will look into this 👍

**Comment by gurrrung** [comment]
> Are you getting this for `graph` command?

This was for `g daily rec` command.

**Comment by baljinderxd** [comment]
> `graph 180` also gives error.

Most probably this is due to the link being generated has more characters than the number of characters allowed in a web link.
I checked and the limit is 2048 characters but `graph 180` generates over 5000 characters. 
It can be fixed using a URL shortener provided by quickchart.io itself.
So should I implement this shortening of the link here itself or would you like to create a new issue and then make a PR for it separately?


> This was for `g daily rec` command.

This is working fine for me. Can you check again and confirm?

**Comment by gurrrung** [comment]
> So should I implement this shortening of the link here itself or would you like to create a new issue and then make a PR for it separately?

Yes, a PR, please.

**Review by gurrrung — APPROVED** [review]
LGTM

**Comment by gurrrung** [comment]
Since this is not an issue related to the graph title I'm merging the current changes.

**Pull request merged**